core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -58,7 +58,7 @@ THE SOFTWARE.
           permission="${it.ARTIFACTS}" />
 
         <j:set var="set" value="${it.changeSet}" />
-        <t:summary icon="symbol-list">
+        <t:summary icon="symbol-changes">
```

Comment:
That doesn't look better either imo, not every change is caused by a PR.
What do you or others think about using `document-text` or `documents` ?